### PR TITLE
Style merge

### PR DESCRIPTION
**Pull request by tim7en**

### CODE REVIEW

**Review by mitchas — APPROVED** [review]
Ran through it once and didn't see any issues. Looks good

File: src/app/modules/core/components/map/map.component.ts [hunk 1]
```diff
@@ -36,12 +36,31 @@ export class MapComponent extends deepCopy implements OnInit, AfterViewInit {
...
 
   public evnt;
   @Input() report: boolean;
+  @Input() mapSize: string;
 
   scaleMap: string;
 
+
+  ngOnChanges(changes: any) {
+	// If map size changed, trigger resize event, which will force map to redraw
```
Comment (tgross-usgs) [inline src/app/modules/core/components/map/map.component.ts:50]: Is a resize event triggered when the footer section pops up? I am unable to see all flowline segments shown in the map when the footer pops up.

File: src/app/modules/core/components/map/map.component.ts [hunk 2]
```diff
@@ -150,9 +172,12 @@ export class MapComponent extends deepCopy implements OnInit, AfterViewInit {
             this.reportMap.remove();
       }
         this.reportMap = new L.Map('reportMap', this.options);
-        // add point of interest
+		// add point of interest
+	  	// MarkerMaker icon
+		var blackPin = L.divIcon({className: 'wmm-pin wmm-black wmm-icon-circle wmm-icon-white wmm-size-25'});
```
Comment (tgross-usgs) [inline src/app/modules/core/components/map/map.component.ts:177]: The new pin looks great but it seems the point of the pin is not directly on the click point. It almost looks as the though the pin is centered on the click point???

**Review by tgross-usgs — APPROVED** [review]
This is looking really great (style-wise and with the new functionality) and may be fine for now but I do notice a few things.

1. The map extent does not seem to change or change enough when the footer section pops up. I lose sight of part of the spill route
2. The table with header names/descriptions in the report seems too small, the contained text is cut off
3. Could the map in the report be made wider to fit the width of the modal?
4. Can we add some type of alert to let the user know that there is a water body cutting off the full distance of their spill search?  I realize the colors of the flowline change but some type of alert or disclaimer would be helpful.

**Pull request merged**